src/hearings/converters/additional-security.converter.ts

```diff
@@ -0,0 +1,22 @@
+import {Store} from '@ngrx/store';
+import {Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+import * as fromHearingStore from '../store';
+import {AbstractConverter} from './abstract.converter';
+
+export class AdditionalSecurityConverter extends AbstractConverter {
+  constructor(protected readonly hearingStore: Store<fromHearingStore.State>) {
+    super(hearingStore);
+  }
+
+  public transformAnswer(): Observable<string> {
+    return this.hearingState.pipe(
+      map(state => {
+        const result = state.hearingRequest.hearingRequestMainModel.caseDetails.caseAdditionalSecurityFlag ?
+          'Yes' : 'No';
```

Comment:
could return directly

Reply:
Personally prefer to separate when it is a long statement like this. The other one like this I agree